tests/e2e/app-files/hello.spec.js

```diff
@@ -3,6 +3,14 @@ describe('Say Hello', () => {
     cy.visit('/');
   });
 
+  it('should save trace log in log.txt', () => {
+    const filePath = '../log.txt';
+
+    cy.readFile(filePath, { timeout: 1000 })
+      .then(fileContents => {
+        expect(fileContents).to.match(/^TRACE.+This is a test\r?\n/);
+      });
+  });
```

Comment:
1. A smaller, custom timeout will speed up the test if the file is not created as expected, but do we *need* it? Simpler code is easier to read and maintain :slightly_smiling_face: 
2. We can just match the beginning of the file and not worry about line endings.

```suggestion
    cy.readFile(filePath)
      .then(fileContents => {
        expect(fileContents).to.match(/^TRACE.+This is a test/);
      });
  });
```